text/0010-gatsby-docs-localization.md

```diff
@@ -0,0 +1,212 @@
+* Start Date: 2019-09-11
+* RFC PR:
+* Gatsby Issue:
+
+
+## Summary
+
+Begin the process of localizing the documentation of Gatsby into non-English languages.
+
+
+## Motivation
+
+[You Belong Here.](https://www.gatsbyjs.org/blog/2018-09-07-gatsby-values/#you-belong-here)
+
+Gatsby believes that everyone belongs in Open Source, and that includes developers that do not speak English. Having access to translated documentation opens up the ability to create websites to people who don't speak English and don't have the privileges to be able to learn.
+
+
+## Detailed Design
+
+The translations will be done in the [same monorepo](https://github.com/gatsbyjs/gatsby) as the rest of the Gatsby documentation and code. Additional tooling, automation, and documentation will be made to ensure that the languages stay up-to-date and to maintain translation quality.
+
+### Overview of Changes
+
+The following things need to be created:
+
+* [ ] A bot to track changes to the English docs and create pull requests
+* [ ] Script to aid in verifying translation accuracy
+* [ ] Script to set up a new language
+* [ ] Documentation for how to start a new language translation and how to contribute translations
+* [ ] A prioritized list of pages to translate
+
+There are several changes to the repo that need to be made in order to support localization:
+
+* [ ] All text strings in `/www` need to be moved to `/docs` so they can be translated
+* [ ] All content in `/docs` need to be moved to `/docs/en`
```

Comment:
Can anyone tell me how much effort this would take? Maybe we could make a different top-level directory instead for each translation (e.g. `docs-es`, `docs-fr`)

Reply:
I don't have experience for the first point about all the other text strings...

for the second point is not much, I just did it [here](https://github.com/horacioh/gatsby/pull/30). I decided to just leave blank the other translations folders and when someone wants to translate the process would be:

* copy files/folders to translate from `docs/en` to `docs/<LANG>`
* translate the documents
* submit a PR

Reply:
There are couple of considerations here that are worth thinking through:
 - movings docs might cause extra work needed for in-progress documentation pull requests (git should handle this gracefully, as it doesn't track files, it tracks content, but I found that it sometimes cause weird issues)
 - from discoverability perspective I think leaving `docs` "as-is" and adding separate directory for translations signals that "source of truth" is english documentation, so if there are any meaningful changes to documentation they should happen there first and later should be handled in translations (and this is flow that would be supported by automation we will need to build)

Reply:
@pieh thanks for your insight! We'll go with that then: keep `docs/` the same and then add a `translations` folder. I'll change the RFC to reflect that.

Reply:
I hope my comment didn't read that we shouldn't do what was proposed - just something that's worth thinking about - I'm definitely not expert on this matter!

Reply:
No, I think you made a good point and I was already starting to think the same as you too.